elasticdl/python/master/master.py

```diff
@@ -220,6 +222,7 @@ def prepare(self):
             self.instance_manager.update_status(InstanceManagerStatus.PENDING)
             if self.distribution_strategy == DistributionStrategy.ALLREDUCE:
                 # Exposes the consensus service for allreduce-based training
+                self.rendezvous_server.start()
```

Comment:
When will we close it?

Reply:
We can close it when the job finishes. However, the master will exit if the job finishes, so we don't need to explicitly close it.